index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <!--start:header-->
+        <header>
+            <nav class="nav-container">
+                <a class="logo" href="#"><img src="images/My Logo.svg" alt="My Logo"></a>
+                <a class="hamburger" href="#"><img src="images/hamburger.svg" alt="hamburger"></a>
+            </nav>
+        </header>
+        <!--End:header-->
+        <!--Start:Section-->
+        <section class="main">
+            <div class="main-section">
+                <h1 class="intro-name">I’m Anshuman.<br>Glad to see<br> you!</h1>
+                <p class="intro-bio">I’m a software developer! I can help you <br>
+                     build a product , feature or website Look<br> through some of my work and<br>
+                      experience! If you like what you see and<br>
+                       have a project you need coded, don’t<br>
+                        hesitate to contact me.</p>
+                <h3 class="social-text">LET’S CONNECT</h3>
+                <div class="social-media">
+                    <div class="social-icon">
+                        <a href="#">
+                            <img src="images/twitter.svg" alt="twitter">
+                        </a>
+                    </div>
+                    <div class="social-icon">
+                        <a href="">
+                            <img src="images/Linkedin.svg" alt="Linkedin">
+                        </a>
+                    </div>
+                    <div class="social-icon">
+                        <a href="">
+                            <img src="images/microverse.svg" alt="Meta">
+                        </a>
+                    </div>
+                    <div class="social-icon">
+                        <a href="">
+                            <img src="images/github.svg" alt="github">
+                        </a>
+                    </div>
+                    <div class="social-icon">
+                         <a href="">
+                                <img src="images/victory.svg" alt="victory">
+                            </a>
+                    </div>
```

Comment:
- [ ] Great job so far @anshuman7negi your project looks great. There are just a few changes you need to make and you'll be on your way to the next project. :fire: 
- [ ] In this part of your project, since these are a list of social icons, I will suggest you make use of the `ul` and `li` tags instead to group them. You can read more about them[ here ](https://www.freecodecamp.org/news/html-list-how-to-use-bullet-points-ordered-and-unordered-lists/) :100: 
- So your code will look something like this: :smile: 

```
<ul class="social-media">
    <li class="social-icon">
        <a href="#">
            <img src="images/twitter.svg" alt="twitter">
        </a>
    </li>
    <li class="social-icon">
        <a href="">
            <img src="images/Linkedin.svg" alt="Linkedin">
        </a>
    </li>
    <li class="social-icon">
        <a href="">
            <img src="images/microverse.svg" alt="Meta">
        </a>
    </li>
    <li class="social-icon">
        <a href="">
            <img src="images/github.svg" alt="github">
        </a>
    </li>
    <li class="social-icon">
        <a href="">
            <img src="images/victory.svg" alt="victory">
        </a>
    </li>
</ul>

```